nx/test/nx/lin_alg_test.exs

```diff
@@ -339,52 +339,83 @@ defmodule Nx.LinAlgTest do
              |> round(3) == round(v, 3)
     end
 
-    # TO-DO investigate why the property test fails
-    # even though we have working tests
-    @tag :skip
     test "property" do
-      for _ <- 1..10 do
-        square = Nx.random_uniform({4, 4})
-
-        assert {u, d, vt} = Nx.LinAlg.svd(square)
-        m = u |> Nx.shape() |> elem(1)
-        n = vt |> Nx.shape() |> elem(0)
-
-        assert_all_close(
-          u
-          |> Nx.dot(diag(d, m, n))
-          |> Nx.dot(vt),
-          square
-        )
-
-        tall = Nx.random_uniform({4, 3})
-
-        assert {u, d, vt} = Nx.LinAlg.svd(tall)
-        m = u |> Nx.shape() |> elem(1)
-        n = vt |> Nx.shape() |> elem(0)
-
-        assert_all_close(
-          u
-          |> Nx.dot(diag(d, m, n))
-          |> Nx.dot(vt),
-          tall
-        )
-
-        # TODO: SVD does not work for wide matrices and
-        # raises a non-semantic error
-
-        #  wide = Nx.random_uniform({3, 4})
-
-        # assert {u, d, vt} = Nx.LinAlg.svd(wide)
-        # m = u |> Nx.shape() |> elem(1)
-        # n = vt |> Nx.shape() |> elem(0)
+      random_offset = fn -> :rand.uniform() * 0.5 + 0.5 end
```

Comment:
@josevalim as per the description on the PR, I'm thinking that we should just remove this test entirely instead. WDYT?
We could open an issue for a better implementation of SVD instead, since it seems ours has convergence problems

Reply:
I would remove the test and call it a day, especially as I plan to work on optional callbacks soon. We have a barebones implementation. If you want performance and accuracy, use one of the backends.